be/src/storage/tablet_manager.cpp

```diff
@@ -1762,4 +1768,16 @@ std::unordered_map<TTabletId, vector<pair<uint32_t, uint32_t>>> TabletManager::g
     return ret;
 }
 
+void TabletManager::_add_shutdown_tablet_unlocked(int64_t tablet_id, DroppedTabletInfo&& drop_info) {
+    auto iter = _shutdown_tablets.find(tablet_id);
+    if (iter != _shutdown_tablets.end() && (iter->second).tablet != nullptr) {
+        // clear meta for the previous shutdown tablet
+        if (auto st = _remove_tablet_meta((iter->second).tablet); !st.ok()) {
+            LOG(FATAL) << "Fail to remove previous table meta, id: " << tablet_id << " status: " << st;
```

Comment:
FATAL will cause BE crash, I don't it is necessary.

Reply:
updated